assets/scripts_ssh_config

```diff
@@ -1,3 +1,4 @@
 StrictHostKeyChecking no
 UserKnownHostsFile /dev/null
+PasswordAuthentication no
```

Comment:
oh duh - good point! I'm building 4.9.31 with your PR atm, and once its published, I'll update mater to use it too - after that, rebasing will re-run the tests with your fix.